openstack/cronus/templates/rhea/secrets.yaml

```diff
@@ -11,5 +11,11 @@ metadata:
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-  config.yaml: {{ include "cronus/templates/rhea/_config.yaml.tpl" . | b64enc }}
-{{- end -}}
+  RABBITMQ_URI: {{ printf "amqp://%s:%s@%s/" .Values.rhea_rabbitmq.users.default.user .Values.rhea_rabbitmq.users.default.password .Values.rhea_rabbitmq.host | b64enc }}
+  {{- if .Values.config.keystone }}
```

Comment:
Same here... can rhea work without a keystone password? And can it work without a client_cert_ou?